src/common/color.helper.ts

```diff
@@ -87,7 +89,7 @@ export class ColorHelper {
     }
   }
 
-  getLinearGradientStops(value, start) {
+  getLinearGradientStops(value, start?) {
```

Comment:
`getLinearGradientStops` was being used with only one argument in `series-vertical.component.ts`. You might want to check that.